agents/registry/registry.py

```diff
@@ -76,13 +84,22 @@ def __init__(self, agent):
             options={'match': 'wildcard'}
         )
 
+        agg_params = {
+            'frame_length': 30*60,
```

Comment:
This is the longest frame length in any Agent. Did we make a formal recommendation of 10 minute frames? All other Agents either have 10 or 1 minute frames.

Reply:
I'll change this to 1 min.